cfme/containers/container.py

```diff
@@ -59,21 +61,45 @@ def get_random_instances(cls, provider, count=1, appliance=None):
                 for obj in itertools.islice(containers_list, count)]
 
 
+class ContainerAllView(BaseLoggedInPage):
+
+    containers = Table(locator="//div[@id='list_grid']//table")
+
+    @View.nested
+    class filters(Accordion):  # noqa
+
+        ACCORDION_NAME = "Filters"
+
+        @View.nested
+        class navigation(VerticalNavigation):
+
+            DIV_LINKS_MATCHING = './/div/ul/li/a[contains(text(), {txt})]'
+
+            def __init__(self, parent, logger=None):
+                VerticalNavigation.__init__(self, parent, '#Container_def_searches', logger=logger)
+        tree = ManageIQTree()
```

Comment:
Do you have a tree and vertical navigation here? I thought it was just the vertical nav elements.